packages/catalog-generator/scripts/build-catalogs.mjs

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ts-node
+// @ts-check
+
+import { spawn } from 'node:child_process';
+import { createRequire } from 'node:module';
+import { dirname } from 'node:path';
+import { resolve } from 'path';
+import { existsSync } from 'node:fs';
+
+const require = createRequire(import.meta.url);
+
+/**
+ * @type {Record<import('../dist/types').CatalogRuntime, string[]>}
+ **/
+const CATALOGS = {
+  Main: ['4.8.1', '4.4.4', '4.4.0.redhat-00046'],
+  Quarkus: ['3.16.0', '3.8.3', '3.8.0.redhat-00014'],
+  SpringBoot: ['4.8.1', '4.4.4', '4.4.0.redhat-00039'],
+};
+const KAMELETS_VERSION = '4.8.1';
```

Comment:
These are the versions that get in the catalog, if we want them to be even friendlier for people to update, we could read them from a `.txt` file instead.